app/controllers/amend/ChangeRegistrationController.scala

```diff
@@ -120,10 +124,19 @@ class ChangeRegistrationController @Inject()(
   }
 
   private def getPreviouslyRegisteredRows(waypoints: Waypoints, sourcePage: CheckAnswersPage)
-                                         (implicit request: AuthenticatedMandatoryIossRequest[_]): Seq[Option[SummaryListRow]] = {
-    val previousRegistrationSummaryRow = PreviousRegistrationSummary.checkAnswersRow(request.userAnswers, Seq.empty, waypoints, sourcePage)
+                                         (implicit request: AuthenticatedMandatoryIossRequest[AnyContent]): Seq[Option[SummaryListRow]] = {
+
+    val previousRegistrationSummaryRow = PreviousRegistrationSummary.checkAnswersRow(
+      answers = request.userAnswers,
+      existingPreviousRegistrations = PreviousRegistration.fromEtmpPreviousEuRegistrationDetails(request.previousEURegistrationDetails),
```

Comment:
Factory method fromEtmpPreviousEuRegistrationDetails converts to our format from ETMPs format.